packages/react/src/components/Tabulation/index.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+class Tabulation extends Component {
```

Comment:
What about renaming this component in `Tabs`?

Reply:
I'm in favour of that :)
👍